docs/decisions/0003-name-affirmation-signals.rst

```diff
@@ -0,0 +1,38 @@
+3. Using Signals for Name Affirmation
+-------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+A `VerifiedName` object has a `status` field that should correspond to the status of the associated IDV or proctored exam attempt.
```

Comment:
Nit: I know you qualify this first line with the last line in the `Context` section, but switching "correspond to" to "is affected by" might be clearer. "correspond to" reads to me as "be equal to", and it's not clarified that the statuses aren't equal until the end of the paragraph.